packages/fhir-converter/src/templates/cda/ValueSet/SystemReference.hbs

```diff
@@ -1,10 +1,16 @@
 {{#if code}}
-    {{#if (eq code '2.16.840.1.113883.6.1')}}
-	    "http://loinc.org"
+    {{#if (startsWith code '1.2.840.114350')}}
+        "https://fhir.epic.com"
+    {{else if (startsWith code '2.16.840.1.113883.3.247')}}
+        "https://terminology.hl7.org/CodeSystem-IMO.html"
```

Comment:
please remove all https URLs - it's weird, but they should always have http only